absl/base/config.h

```diff
@@ -397,6 +397,7 @@ static_assert(ABSL_INTERNAL_INLINE_NAMESPACE_STR[0] != 'h' ||
 //   AsmJS                             __asmjs__
 //   WebAssembly                       __wasm__
 //   Fuchsia                           __Fuchsia__
+//   VxWorks                           __VXWORKS__
```

Comment:
Please remove this. This is not a supported platform. Support for this platform is best-effort.

Reply:
Thank you for your quick review.  I have changed source code to solve it.